### ISSUE TITLE
Annotated variable names that begin with class emit bad code

### ISSUE THREAD

**Issue by euresti** [hhatto/autopep8#365]
This is related to https://github.com/PyCQA/pycodestyle/issues/716

```
class Foo:
    classification: int
    definition: int
    _def: int
    _class: int


def foo():
    class_foo: int
    def_value: int
```

results in 
```
class Foo:
    classification:
        int
    definition: int
    _def: int
    _class: int


def foo():
    class_foo:
        int
    def_value: int
```

Weird that `def` isn't affected.

**Comment by hhatto** [comment]
see also https://github.com/PyCQA/pycodestyle/issues/682

**Comment by ErikBjare** [comment]
Just happened for me with a variable named `classes`.

**Comment by asottile** [comment]
I believe this got fixed in pycodestyle and can be closed:

```console
$ autopep8 t.py 
class Foo:
    classification: int
    definition: int
    _def: int
    _class: int

```

**Comment by hhatto** [comment]
This problem no longer occurs from version 1.4.1.

fix in 95832804e361588cb8e15d848a20089fbd8d9029.

> update requirement, pycodestyle 2.4.0 and higher version
